wazimap_ng/boundaries/views.py

```diff
@@ -33,24 +35,29 @@ def get_geography(self, code):
 
 @cache_decorator("geography_item")
 def geography_item_helper(code, version):
-    geography = get_object_or_404(Geography, code=code, version=version)
-    serializer = serializers.GeographyBoundarySerializer(geography.geographyboundary)
+    geography = get_object_or_404(Geography, code=code)
+    boundary = geography.geographyboundary_set.filter(version=version).first()
```

Comment:
Is this relying on boundary geography and version pair to be unique together? Perhaps worth a comment.
Alternatively
Boundary.get(version=version, geography=geography) which would raise if there's more than one